typical_bezeq_run.log

```diff
@@ -115,11 +103,11 @@
 2023-10-31 14:45:01.107 notifications-filter      (D) Initialization started.
 2023-10-31 14:45:01.108 notifications-filter      (I) Filter config file:/local/veego/agent/notifications-filter-config.json
 2023-10-31 14:45:01.108 notifications-filter      (D) Loading notification filters from file: /local/veego/agent/notifications-filter-config.json
-2023-10-31 14:45:01.108 notifications-filter      (V) missing: discovery-malfunctions
-2023-10-31 14:45:01.108 notifications-filter      (V) missing: router-malfunctions
-2023-10-31 14:45:01.108 notifications-filter      (V) missing: router-general-malfunctions
-2023-10-31 14:45:01.109 notifications-filter      (V) missing: wifi-device-malfunctions
-2023-10-31 14:45:01.109 notifications-filter      (V) missing: wifi-interface-malfunctions
+2023-10-31 14:45:01.108 notifications-filter      (W) missing: discovery-malfunctions
```

Comment:
"missing" seems like it's something we were expecting to be there so warning seems more suitable
One-liner?